### DISCUSSION TITLE
Why can resource-groups.refresh-interval only be set in db？

### DISCUSSION THREAD

**Discussion by A-little-bit-of-data** [trinodb/trino#24309] [Q&A]
Why can resource-groups.refresh-interval only be set in db？
resource-groups.configuration-manager=file Why can't I configure a security.refresh-period=1s like access-control.name=file

**Comment by hashhar** [comment]
simply because it's not implemented (and it's non-trivial).

The RG structure is a tree and there are issues when the tree structure changes and some queries are already running against the old structure.

**Reply by A-little-bit-of-data** [reply]
Thank you very much for your answer. I would like to ask if there is any plan to implement it in the future?

**Reply by hashhar** [reply]
No, if you want dynamic RG config you should use the database based implementation.

That was the entire reason the database based RG config was added.
